### ISSUE TITLE
PESQ Trending Questions

### ISSUE THREAD

**Issue by a897456** [sp-uhh/sgmse#74]
I found that the trend of PESQ has been increasing. Does that mean that a larger epoch will get a higher PESQ? In this case, is there no upper limit? Why does Reference 2 use 160 epochs? Is it because PESQ will converge after 160 epochs? I am currently at 151 epochs, but PESQ has no trend of convergence, as shown in the figure below

![Image](https://github.com/user-attachments/assets/59eb2b65-43fe-45f7-83aa-fecf548a2593)

**Comment by a897456** [comment]
I tested the models with three training steps of 200k, 300k and 450k, and found that their PESQs were 3.5, 3.47 and 3.42 respectively. That is to say, although the PESQ curve keeps rising during training, the PESQ of the test dataset keeps decreasing. What is going on? It seems to indicate that the PESQ curve during training cannot reflect the performance of the model. This makes it more difficult for me to choose the right epoch.

**Comment by julius-richter** [comment]
This could be an overfitting effect. From the issue, it is also not entirely clear which dataset/setup you used, so it is hard to put the reported metric values into context.

For VoiceBank-DEMAND, we observed saturation without clear overfitting, which we verified using the validation set.

In general, the training PESQ curve does not necessarily reflect test-set performance. As your experiments show, longer training can improve training PESQ while slightly degrading PESQ on unseen data.

I will close this issue due to inactivity.
